<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="true" />
 <!--<env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find gazebo_ardu)/launch/custom_rosconsole.conf"/>-->
    <!--<arg name="fcu_url" default="udp://:14550@"/>-->
    <arg name="fcu_url" default="tcp://127.0.0.1:5772"/>
    <arg name="gcs_url" default=""/>
    <arg name="tgt_system" default="2"/>
    <arg name="tgt_component" default="1"/>
    <arg name="log_output" default="screen"/>
    
    <include file="$(find mavros)/launch/node.launch">
        <arg name="pluginlists_yaml" value="$(find patrol_sim)/launch/apm_pluginlists.yaml"/>
        <arg name="config_yaml" value="$(find patrol_sim)/launch/apm_config.yaml"/>

        <arg name="fcu_url" value="$(arg fcu_url)"/>
        <arg name="gcs_url" value="$(arg gcs_url)"/>
        <arg name="tgt_system" value="$(arg tgt_system)"/>
        <arg name="tgt_component" value="$(arg tgt_component)"/>
        <arg name="log_output" value="$(arg log_output)"/>
    </include>
    <!--<node pkg="patrol_sim" type="patroler.py" name="patroler" output="screen">-->
        <!--<param name="vehicle_name" value="iris_demo1"/>-->
    <!--</node>-->
</launch>

